### PR TITLE
make transparent what is community maintained

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG_REPORT.yml [hunk 1]
```diff
@@ -62,13 +62,14 @@ body:
         Select installation method you've used.
         _Describe the method in the "Additional info" section if you chose "Other"._
       options:
-        - "Manual installation"
```
Comment (artonge) [inline .github/ISSUE_TEMPLATE/BUG_REPORT.yml:65]: Removed ?

Comment (szaimen) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.yml:65]: nope, moved a bit further down...

Comment (szaimen) [reply .github/ISSUE_TEMPLATE/BUG_REPORT.yml:65]: "Community Manual installation with Archive"

**Review by artonge — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Comment by szaimen** [comment]
failing test unrelated

**Pull request merged**